src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -80,29 +104,21 @@ function useCounterparty (agentId) {
   return { holofuelCounterparty, loading }
 }
 
-function useTransactionsWithCounterparties () {
-  const { data: { holofuelUser: whoami = {} } = {} } = useQuery(HolofuelUserQuery)
-  const { data: { holofuelInboxCounterparties = [] } = {} } = useQuery(HolofuelInboxCounterpartiesQuery, { fetchPolicy: 'cache-and-network' })
-  const { data: { holofuelActionableTransactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'cache-and-network' })
-  const { data: { holofuelNonPendingTransactions = [] } = {} } = useQuery(HolofuelNonPendingTransactionsQuery, { fetchPolicy: 'cache-and-network' })
-
-  const updateCounterparties = (transactions, counterparties) => transactions.map(transaction => ({
-    ...transaction,
-    counterparty: counterparties.find(counterparty => counterparty.id === transaction.counterparty.id) || transaction.counterparty
-  }))
-
-  const allCounterparties = uniqBy('id', holofuelInboxCounterparties.concat([whoami]))
+function useUpdatedTransactionLists () {
```

Comment:
I don't think we need this hook anymore. Better to have one hook for each query, that is responsible for whatever clean up we want to do. 

But that update can be part of the chore I suggested over here
<-----